docs/quickstart.rst

```diff
@@ -117,35 +135,52 @@ we can evaluate metrics for subgroups within the data as below:
     >>> from fairlearn.metrics import MetricFrame
     >>> from sklearn.metrics import accuracy_score
     >>> from sklearn.tree import DecisionTreeClassifier
+    >>> from sklearn.model_selection import train_test_split
+    >>>
+    >>> X_train, X_test, y_train, y_test, \
+    A_train, A_test = train_test_split(X_ohe, y, race, random_state=123)
     >>>
     >>> classifier = DecisionTreeClassifier(min_samples_leaf=10, max_depth=4)
-    >>> classifier.fit(X, y_true)
+    >>> classifier.fit(X_train, y_train)
     DecisionTreeClassifier(...)
-    >>> y_pred = classifier.predict(X)
-    >>> mf = MetricFrame(metrics=accuracy_score, y_true=y_true, y_pred=y_pred, sensitive_features=sex)
+    >>> y_pred = (classifier.predict_proba(X_test)[:,1] >= 0.1)
+    >>> mf = MetricFrame(metrics=accuracy_score, y_true=y_test, y_pred=y_pred, \
+    sensitive_features=A_test)
     >>> mf.overall
-    0.8443...
+    0.529185173538776
     >>> mf.by_group
-    sex
-    Female    0.9251...
-    Male      0.8042...
+    race
+    AfricanAmerican    0.542303
+    Asian              0.690323
+    Caucasian          0.520261
+    Hispanic           0.595528
+    Other              0.561798
+    Unknown            0.591525
     Name: accuracy_score, dtype: float64
 
-Additionally, Fairlearn has lots of other standard metrics built-in, such as
+Note that above, because the outcome variable exhibits label imbalance, we set 
+the threshold for triggering a positive prediction lower 
+(to a probability of 0.1 or greater).
```

Comment:
I like that you made this adjustment for imbalance but the number 0.1 kind of falls from the sky...

Did you try a few and decide on that or where is it from? 

I checked if DTC has anything for changing the objective, but all I found was the "balanced" class_weight arg https://scikit-learn.org/stable/modules/generated/sklearn.tree.DecisionTreeClassifier.html
I've never used it but perhaps it could help here?

Adding other @fairlearn/fairlearn-maintainers to chime in if they have ideas.

Reply:
Yeah, I understand! The proportion of positive labels overall was about 11% so I rounded down, but happy to change it to whatever everyone deems would be a better fit.

Reply:
Is it possible to just mention that it's set based on the percentage of positive labels in the dataset? That way, it doesn't just occur out of nowhere 🙂

Reply:
Done! Added a sentence: "We chose this threshold to match the percentage of positive labels in the dataset."